### ISSUE TITLE
Sahi CLI requires pycocotools and terminaltables

### ISSUE THREAD

**Issue by weiji14** [obss/sahi#220]
Running a simple `sahi --help` on the command line doesn't work without `pycocotools` and/or `terminaltables` installed, even though they are not listed as required depedencies in [`requirements.txt`](https://github.com/obss/sahi/blob/0.8.1/requirements.txt) (`sahi` v0.8.1). Reported at https://github.com/conda-forge/staged-recipes/pull/16106#issuecomment-915235141.

Without `pycocotools`:

```python-traceback
$ sahi --help
Traceback (most recent call last):
  File "/home/username/miniconda3/envs/temp/lib/python3.8/site-packages/sahi/scripts/coco_error_analysis.py", line 11, in <module>
    from pycocotools.coco import COCO
ModuleNotFoundError: No module named 'pycocotools'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/username/miniconda3/envs/temp/bin/sahi", line 5, in <module>
    from sahi.cli import app
  File "/home/username/miniconda3/envs/temp/lib/python3.8/site-packages/sahi/cli.py", line 7, in <module>
    from sahi.scripts.coco_error_analysis import main as coco_error_analysis
  File "/home/username/miniconda3/envs/temp/lib/python3.8/site-packages/sahi/scripts/coco_error_analysis.py", line 14, in <module>
    raise ImportError('Please run "pip install -U pycocotools" ' "to install pycocotools first for coco evaluation.")
ImportError: Please run "pip install -U pycocotools" to install pycocotools first for coco evaluation.
```

Without `terminaltables`:

```python-traceback
Traceback (most recent call last):
  File "/home/username/miniconda3/envs/temp/bin/sahi", line 5, in <module>
    from sahi.cli import app
  File "/home/username/miniconda3/envs/temp/lib/python3.8/site-packages/sahi/cli.py", line 8, in <module>
    from sahi.scripts.coco_evaluation import main as coco_evaluation
  File "/home/username/miniconda3/envs/temp/lib/python3.8/site-packages/sahi/scripts/coco_evaluation.py", line 10, in <module>
    from terminaltables import AsciiTable
ModuleNotFoundError: No module named 'terminaltables'
```

Temporary workaround for users would be to `pip install pycocotools terminaltables`. But better solution would be to ether:

1. List these as required depedencies in `requirements.txt`
2. Fix the scripts so that these two packages are fully optional.

**Comment by fcakyon** [comment]
@weiji14 thanks for the detailed reports! if tests pass, will merge into main: https://github.com/obss/sahi/pull/221

**Comment by fcakyon** [comment]
@weiji14 it should be fixed with v0.8.2, would you try again?
